src/djcytoscape/forms.py

```diff
@@ -1,28 +1,21 @@
 from django import forms
-
-from queryset_sequence import QuerySetSequence
+from django.contrib.contenttypes.models import ContentType
 
 from utilities.forms import FutureModelForm
-from utilities.fields import ContentObjectChoiceField
-from utilities.widgets import ContentObjectSelect2Widget
+from utilities.fields import AllowedContentObjectChoiceField as ContentObjectChoiceField
 
 from .models import CytoScape
 
 
-def get_model_options():
-    """
-        provides ContentTypes that are part of CytoScape.ALLOWED_INITIAL_CONTENT_TYPES
-        formatted for utilities.fields.ContentObjectModelField use
-        from prerequisites.forms:
-            Can't always dynamically load this list due to accessing contenttypes too early
-            So instead provide a hard coded list which is checked during testing to ensure it matches
-            what the dynamically loaded list would have produced
-    """
-    from quest_manager.models import Quest
-    from badges.models import Badge
-    from courses.models import Rank
+class AllowedContentObjectChoiceField(ContentObjectChoiceField):
```

Comment:
call this `CytoscapeContentObjectChoiceFiled` since that's what it is, specific to Cytoscape model.